src/js/services/apiservice.js

```diff
@@ -0,0 +1,39 @@
+export const RegisterUser = (username, surname, passsword, email, phoneNumber, membershipType)=>{
+
+    fetch('http://ereader.retrotest.co.za/api/Db', {
+    method: 'POST',
+    headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+        "name": username,
+        "surname": surname,
+        "email": email,
+        "phoneNumber": phoneNumber,
+        "password": passsword,
+        "membershipType": membershipType
+        })
+    });
+}
+
+export const LoginUser = (username, pass)=>{
```

Comment:
You can keep in your diary if you just want to remember.
Only useful functions should be declared.

Reply:
This has been updated